clicache/integration-test/CMakeLists.txt

```diff
@@ -103,10 +103,9 @@ set_property(TEST OverflowTestsN PROPERTY LABELS OMITTED)
 set_property(TEST ThinClientDeltaTestN PROPERTY LABELS OMITTED)
 set_property(TEST ThinClientDurableTestsN PROPERTY LABELS OMITTED)
 set_property(TEST ThinClientHARegionTestsN PROPERTY LABELS OMITTED)
-set_property(TEST ThinClientRegionTestsN PROPERTY LABELS OMITTED)
 set_property(TEST ThinClientSecurityAuthTestsMUN PROPERTY LABELS OMITTED)
 set_property(TEST ThinClientSecurityAuthTestsN PROPERTY LABELS OMITTED)
-set_property(TEST ThinClientSecurityAuthTestsMUN PROPERTY LABELS OMITTED)
+set_property(TEST RegionFailingTests PROPERTY LABELS OMITTED)
```

Comment:
Did we introduce new failing tests and mark them as omitted as part of this changeset?

Reply:
If you look at the test names, you should be able to see them line up with what use to be in the `ThinClientRegionTestsN` file.

Reply:
Got it, just making sure.  I agree this is an improvement.